.github/workflows/grpc.yml

```
@@ -0,0 +1,67 @@
+name: grpc-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  grpc-testsuite-1_37_0:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: grpc
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: python -m pip install --upgrade pip setuptools wheel
```

```suggestion
        run: pip install --upgrade pip setuptools wheel
```
Let me know if running without explicitly calling `python -m` resolves the breaking steps.

---

Same issue unfortunately 😭